dunstrc

```diff
@@ -198,6 +198,9 @@
     # automatically after a crash.
     startup_notification = false
 
+    # window corners
```

Comment:
Please add this stuff to the manpage, too and elaborate it a bit more and give a unit. For a passive user, this could be anything (em, inch, pixels).